w_common/example/disposable/example.dart

```diff
@@ -32,28 +32,31 @@ void main() {
 
   Disposable.enableDebugMode();
 
-  ButtonElement createButton = querySelector('#create-button');
-  ButtonElement disposeButton = querySelector('#dispose-button');
-  InputElement childCountField = querySelector('#child-count-field');
-  InputElement treeDepthField = querySelector('#tree-depth-field');
+  ButtonElement createButton = querySelector('#create-button') as ButtonElement;
+  ButtonElement disposeButton =
+      querySelector('#dispose-button') as ButtonElement;
+  InputElement? childCountField =
+      querySelector('#child-count-field') as InputElement?;
+  InputElement? treeDepthField =
+      querySelector('#tree-depth-field') as InputElement?;
```

Comment:
This don't need to be nullable. We could be able to find the element every time.
```suggestion
  InputElement childCountField =
      querySelector('#child-count-field') as InputElement?;
  InputElement treeDepthField =
      querySelector('#tree-depth-field') as InputElement?;
```